### ISSUE TITLE
Mac build signing?

### ISSUE THREAD

**Issue by FuzzyBearBTC** [SigmundAlpha/reforge#8]
Just opened peerunityd and the message "peerunityd" can't be opened because it is from an unidentified developer.  I know how to change my local settings to allow this, but I am concerned on how you will sign the mac build in this build process. 

Hopefully you have this all figured out already but something worth considering now if you have not.

Fuzzybear

**Comment by SigmundAlpha** [comment]
Nope, I completely forgot to sign it.  I'll look into making sure the next one is signed.  Thanks for the heads up.
